src/github.ts

```diff
@@ -190,4 +195,84 @@ export class GitHub {
       url: artifactURL
     };
   }
+
+  public static async writeBuildSummary(opts: BuildSummaryOpts): Promise<void> {
+    // can't use original core.summary.addLink due to the need to make
+    // EOL optional
+    const addLink = function (text: string, url: string, addEOL = false): string {
+      return `<a href="${url}">${text}</a>` + (addEOL ? os.EOL : '');
+    };
+
+    const refsSize = Object.keys(opts.exportRes.refs).length;
+
+    // prettier-ignore
+    const sum = core.summary
+      .addHeading('Docker Build summary', 1)
+      .addRaw(`<p>`)
+        .addRaw(`For a detailed look at the build, download the following <code>.dockerbuild</code> file and import it into Docker Desktop's Build views. `)
+        .addBreak()
+        .addRaw(`The build file provides insights of your build, such as timing, dependencies, results, logs, traces, and more. `)
+        .addRaw(addLink('Learn more', 'https://docs.docker.com/desktop/use-desktop/builds/#import-build-records'))
+      .addRaw('</p>')
+      .addRaw(`<p>`)
+        .addRaw(`:arrow_down: ${addLink(`<strong>${opts.uploadRes.filename}</strong>`, opts.uploadRes.url)} (${Util.formatFileSize(opts.uploadRes.size)})`)
+        .addBreak()
+        .addRaw(`This file includes <strong>${refsSize} build record${refsSize > 1 ? 's' : ''}</strong>.`)
+      .addRaw(`</p>`)
+      .addRaw(`<p>`)
+        .addRaw(`Was this useful? `)
+        .addRaw(addLink('Let us know', 'https://docs.docker.com/feedback/desktop-build'))
```

Comment:
This link needs to be changed so it redirects to the GHA summary feedback survey (cc @colinhemmings @dvdksn)